packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-limitations/main/index.md

```diff
@@ -115,4 +115,4 @@ Contact Okta support if the **Edit** button is disabled.
 
 #### Option 2: Inform users
 
-Inform your users to ignore the Verify Email Address link and continue the activation through the code flow.
\ No newline at end of file
+Inform your users to ignore the Verify Email Address link and continue the activation through the code flow.
```

Comment:
Why is this change here? Are limitation changes being done in a separate PR?

Reply:
That's not a change. It's how it is currently. I don't know why github registered it as a change.